src/components/ReactCompanies/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { useState, type ReactNode } from 'react';
+import ReactCompany from '@components/ReactCompany';
+import content from '@content/content';
+import { TAGS_FROM_COMPANIES, ALL_TAG } from '@domain/constants';
+import type { Companies } from '@domain/types';
+import clsx from 'clsx';
+import styles from './ReactCompanies.module.scss';
+
+interface ReactCompaniesProps {
+  title?: ReactNode;
+}
+
+const ReactCompanies = ({ title }: ReactCompaniesProps): JSX.Element => {
+  const [selectedTag, setTag] = useState<string>(ALL_TAG);
+
+  const TagButton = ({ value }: { value: string }): JSX.Element => {
+    return (
+      <button
+        id={value}
+        className={clsx(styles.tagButton, value === selectedTag && styles.selected)}
+        onClick={() => {
+          setTag(value);
+        }}
+      >
+        {value}
+      </button>
+    );
+  };
+
+  return (
+    <section className={styles.companies}>
+      {title}
+      <div className={styles.tags}>
+        <TagButton value={ALL_TAG} />
+
+        {TAGS_FROM_COMPANIES.map((tag: string) => (
+          <TagButton value={tag} />
+        ))}
+      </div>
+
+      <div className={styles.content}>
+        {content.companiesUsingScala
+          .filter(({ tags }) => selectedTag === ALL_TAG || tags.includes(selectedTag))
+          .map(({ image, title, description, tags, imageAlt }: Companies, index) => (
```

Comment:
Jeśli typujemy obiekt (pojedynczy) to lepiej typ/interface nazwać w liczbie pojedynczej -> `Company`. Tak samo z pozostałym typami [tutaj](https://github.com/scala-poland/scala-poland-site-new/blob/feature/conferences-and-compoanies-using-scala-sections/src/domain/types.ts) -> `LearningMaterials`, `MeetupGroups`, `Conferences`. Nazwa w liczbie mnogiej sugeruje tablicę